conveyer/conveyer.py

```diff
@@ -117,38 +118,52 @@ def Conveyer(config, file_override=None, renamer=None):
     return c
 
 
-conveyer = Conveyer(
-    config={"log_file": "/tmp/logs"},
-    file_override=file,
-    renamer=os.rename,
-)
+@attr.s(hash=False)
+class ConveyerApp(object):
+    """Model a single instance of the Conveyer daemon."""
 
-app = Klein()
+    conveyer = attr.ib(validator=attr.validators.instance_of(_Conveyer))
 
+    app = Klein()
 
-@app.route('/')
-def hello(request):
-    """Report if we're still alive and functioning."""
-    request.response = 200
-    return "Still running!\n"
-
-
-@app.route('/log', methods=['POST'])
-def accept_log(request):
-    """Accept a log message for logging."""
-    the_log = request.content.read()
-    conveyer.execute(conveyer.log("{0}\n".format(the_log)))
-    request.response = 200
-    return "ok"
-
-
-@app.route('/rotate', methods=['POST'])
-def rotate_log(request):
-    """Rotate the log, and report the filename back to the client."""
-    fn = conveyer.rotate_logs()
-    request.response = 200
-    return fn
+    def reset(self):
+        """Obtains a Conveyer instance to forward requests to."""
+        assert False
```

Comment:
Does anything call this function yet?  It seems like it'd just raise an exception, and resetting the Conveyer will never actually happen?